openupgrade_scripts/scripts/sale/14.0.1.1/noupdate_changes.xml

```diff
@@ -146,10 +148,4 @@
     <field name="name">Personal Invoices</field>
     <field name="domain_force">[('move_type', 'in', ('out_invoice', 'out_refund')), '|', ('invoice_user_id', '=', user.id), ('invoice_user_id', '=', False)]</field>
   </record>
-  <record id="account_invoice_line_rule_see_all" model="ir.rule">
-    <field name="domain_force">[('move_id.move_type', 'in', ('out_invoice', 'out_refund'))]</field>
-  </record>
-  <record id="account_invoice_line_rule_see_personal" model="ir.rule">
-    <field name="domain_force">[('move_id.move_type', 'in', ('out_invoice', 'out_refund')), '|', ('move_id.invoice_user_id', '=', user.id), ('move_id.invoice_user_id', '=', False)]</field>
-  </record>
```

Comment:
don't remove this due to https://github.com/OCA/OpenUpgrade/commit/17e556d243fd4f7bb8960105ab1c884fbf16c160